src/cordova-plugin-outline/android/scripts/copy_third_party.js

```diff
@@ -20,7 +20,9 @@ module.exports = function(context) {
   console.log('Copying Android third party libraries...');
   child_process.execSync('mkdir -p plugins/cordova-plugin-outline/android/libs');
   child_process.execSync(
-      `cp third_party/outline-go-tun2socks/android/tun2socks.aar plugins/cordova-plugin-outline/android/libs/`);
+    `cp third_party/outline-go-tun2socks/android/tun2socks.aar plugins/cordova-plugin-outline/android/libs/`
```

Comment:
Why is this changing? I think we indent with 4 spaces on line breaks?

Reply:
It's definitely been 2 - I can update it to 4 in the prettierrc in a separate PR